test/lib.search.js

```diff
@@ -64,19 +64,20 @@ describe('Search method', () => {
   });
 
   describe('more results mapping', () => {
-    it('schould return few netflix apps', () => {
+    it('should return few netflix apps', () => {
       return gplay.search({ term: 'netflix' })
         .then((apps) => {
           assert.equal(apps[0].appId, 'com.netflix.mediaclient');
-          assertIdsInArray(apps, 'com.netflix.ninja', 'com.netflix.NGP.StrangerThings');
+          // assert there are at least two apps with '.netflix' in the id
+          assertIdsInArray(apps, 'com.netflix.ninja', 'com.netflix.NGP.Poinpy' || 'com.netflix.NGP.SpongeBobKrustyCookOff' || 'com.netflix.NGP.StrangerThings' || 'com.netflix.NGP.Oxenfree');
```

Comment:
how does this OR (||) work here? I think it's best to not use assertIdsInArray here and assert manually for the condition described in the comment